packages/react-reconciler/src/ReactFiberComponentStack.js

```diff
@@ -59,6 +59,7 @@ function describeFiber(fiber: Fiber): string {
 }
 
 export function getStackByFiberInDevAndProd(workInProgress: Fiber): string {
+  // TODO (brian) Generate and store temporary ID so DevTools can match up a component stack later.
```

Comment:
Oops I missed this. This is the wrong place for this comment. The comment belongs in the user timing mark code, as a note that maybe we don't need to pre-generate stacks in the future.

Reply:
Oh sorry, I misread your earlier comment. I'll move this comment to above the calls to `getStackByFiberInDevAndProd`

Reply:
No worries.